RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 - [*] Plans: Expired or cancelled plans are now shown more reliably [https://github.com/woocommerce/woocommerce-ios/pull/9924]
 - [Internal] Store creation: a workaround was previously implemented that can result in an inaccurate app experience like when the free trial banner is not shown immediately after store creation due to out-of-sync site properties. Now that the API issue is fixed, the app now waits for the site for a bit longer but ensures all necessary properties are synced. [https://github.com/woocommerce/woocommerce-ios/pull/9957]
+- [Internal] Product details AI: Updated prompts to identify the language in provided text to use in responses.
```

Comment:
nit: might be worth mentioning the entry point briefly for easier testing. the PR link would be useful too

Reply:
Updated in fdad27f7dd28871aa85331c4ed1b7d8ba73e9bea.